packages/backend/index.js

```diff
@@ -390,6 +390,83 @@ app.get("/challenges", withRole("admin"), async (request, response) => {
   }
 });
 
+app.post("/api/frame", (req, res) => {
+  const baseUrl = "https://speedrunethereum.com";
+  const BG_SERVER = process.env.BG_BACKEND;
+
+  const challengeButtons = {
+    0: {
+      label: "🎟 Simple NFT Example",
+      link: "https://speedrunethereum.com/challenge/simple-nft-example",
+    },
+    1: {
+      label: "🥩 Staking App",
+      link: "https://speedrunethereum.com/challenge/decentralized-staking",
+    },
+    2: {
+      label: "🏵 Token Vendor",
+      link: "https://speedrunethereum.com/challenge/token-vendor",
+    },
+    3: {
+      label: "🎲 Dice Game",
+      link: "https://speedrunethereum.com/challenge/dice-game",
+    },
+    4: {
+      label: "⚖️ Build a DEX",
+      link: "https://speedrunethereum.com/challenge/minimum-viable-exchange",
+    },
+    5: {
+      label: "📺 State Channel App",
+      link: "https://speedrunethereum.com/challenge/state-channels",
+    },
+  };
+
+  try {
+    const { query } = req;
+
+    let idAsNumber = 0;
+
+    if (query && query.id) {
+      idAsNumber = Number(query.id);
+    }
+
+    const nextId = idAsNumber + 1;
+
+    if (idAsNumber === 6) {
```

Comment:
NOTE: I am stopping at challenge 5 because it doesn't seem like the other two challenges have been ported over yet.

Reply:
Yep, we can add them later